Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "acquire-imaging"
 authors = ["Nathan Clack <nclack@chanzuckerberg.com>"]
-version = "0.1.4"
+version = "0.2.0"
```

Comment:
Should I understand the minor bump here as representing a pre-v1 breaking change to the public API (i.e. because we removed `ChunkingProperties`)?

Should there have been an equivalent minor version bump for acquire-core-libs for the equivalent change there? If so, I don't think we have to do that now, but wanted to check to see if I understand things correctly.